### PR TITLE
Added instructions for Groups

### LINKED ISSUE #114: Write instructions on groups page
- [ ] Users are not clear about the difference between groups and collaborations
- [ ] Students are not understanding that they need to start their assignment on the group page. 
- [ ] Students are not clear about how their assignment will be submitted. 

Some information can be found here https://docs.circuitverse.org/#/groups

### PR DESCRIPTION
**Pull request by JyothsnaAshok**
Fixes #114  

[GCI Task](https://codein.withgoogle.com/dashboard/task-instances/6269829040832512/)
#### Describe the changes you have made in this pr -
I've added instructions for teachers to add members, new assignments and for students on submitting their projects and adding collaborators. Changes have been made to both the specific group page and the users' groups page. 

### Screenshots of the changes (If any) -
image

image

image

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@JyothsnaAshok I think you should document these on https://docs.circuitverse.org/

**Comment by JyothsnaAshok** [comment]
@Ananthavijay I've already done that. The task asked us to include it in the groups page as well. ☺️

**Comment by sakshi1499** [comment]
I think we should also add about collaborations here? Also, if you could add one assignment and see how the alignment is affected of the description you have added. I don't want it to overflow. Can you check that once? @JyothsnaAshok

File: app/views/users/logix/groups.html.erb [hunk 1]
```diff
@@ -2,6 +2,12 @@
   <div class="row feature">
 
     <div class="col">
+         
+         <p> <b>Groups</b> is a versatile feature of CircuitVerse which helps teachers to manage class assignments with ease. Teachers can add students to groups, post assignments with deadlines and also grade the assignments using the built-in preview.</p>
+      <% unless @groups_mentored.present? %>
```
Comment (tachyons) [inline app/views/users/logix/groups.html.erb]: Avoid unless if there are negation method available, like if @groups_mentored.blank?

**Comment by JyothsnaAshok** [comment]
@sakshi1499 Done!

**Comment by sakshi1499** [comment]
Good work! Can you please follow up with what @tachyons has mentioned? 😊

**Comment by JyothsnaAshok** [comment]
@tachyons I've removed the extra white spaces but some had to be kept to follow the style of code of the rest of the file.

**Comment by sakshi1499** [comment]
Looks good to me! 😊

**Comment by tachyons** [comment]
Those was added before we configured erblint using hound, so follow hound suggestions whenever possible

**Comment by tachyons** [comment]
@JyothsnaAshok any updates ?

**Comment by nitin10s** [comment]
@tachyons Please close this?

**Pull request closed without merging**